scripts/ci/provisioning/freebsd.sh

```diff
@@ -29,6 +29,20 @@ if ! pkg info > /dev/null; then
   exit 1
 fi
 
+## Jan 7 2020 - work around image having mangled perl defaults
+if ! pkg remove -y perl5.26; then
```

Comment:
the condition won't work since removing perl5.26 returns 0.

Reply:
I'd rather just replace this whole if with a simple `pkg install -f perl5`, that way we're sure it is installed.

Reply:
> 
> 
> the condition won't work since removing perl5.26 returns 0.

You can *definitely* tell I had a 16 hour day yesterday... brain cut out in the middle of writing it. Should have been `pkg info` check then remove. Updating now.

Reply:
> 
> 
> I'd rather just replace this whole if with a simple `pkg install -f -y perl5`, that way we're sure it is installed.

The problem there is that `pkg install -f -y perl5` may or may not install the correct one, depending on whether or not (and when) ports defaults are changed or updated. Being on 12.0 kind of complicates this since 12.0 goes EoS next month, so being non-specific here might potentially lead to bad or unexpected behavior. (e.g. may not create `/usr/local/bin/perl` or may not function as expected; there's no guarantees on EoS releases.)

Reply:
Maybe just don't use Perl?

Reply:
@jaykrell automake is based on perl so we don't have a choice :)

Reply:
It's actually even worse than that, I'm afraid... Perl is a mandatory dependency for: git, automake, autoconf, curl, and literally 1001 other packages.

Reply:
Autoconf/automake — no problem, use cmake. :)